src/wp-includes/option.php

```diff
@@ -2144,16 +2144,27 @@ function update_network_option( $network_id, $option, $value ) {
 	 */
 	$value = apply_filters( "pre_update_site_option_{$option}", $value, $old_value, $option, $network_id );
 
+	$is_multisite = is_multisite();
 	/*
```

Comment:
```suggestion

         if ( has_filter( "pre_site_option_{$option}" ) ) {
		global $wp_filter;

		$old_filters = $wp_filter[ "pre_site_option_{$option}" ];
		unset( $wp_filter[ "pre_option_{$option}" ] );

		$raw_old_value                       = get_network_option( $network_id, $option, false );
		$wp_filter[ "pre_option_{$option}" ] = $old_filters;
	} else {
		$raw_old_value = $old_value;
	}
	/*
```

Don't we need an unfiltered version of this value?

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5434/commits/2671d63d407d780458741daccfd2122e4dad29ff